sbt/src/server-test/completions/target/streams/test/$global/$global/definedTestNames/data

```diff
@@ -0,0 +1 @@
+["org.sbt.ExampleSpec"]
```

Comment:
this is the trick to avoid compilation in Jenkins + DBuild.
cc. @eed3si9n @cunei